AirCasting/SearchAndFollow/CompleteScreen/Views/SearchCompleteScreenMapView.swift

```diff
@@ -25,6 +25,7 @@ struct SearchCompleteScreenMapView: UIViewRepresentable {
         
         let camera = GMSCameraPosition.camera(withLatitude: latitude, longitude: longitude, zoom: 15.0)
         let mapView = GMSMapView.map(withFrame: .zero, camera: camera)
+        if Constants.isSatelliteMapOn { mapView.mapType = .satellite }
```

Comment:
As I mentioned, I'm not a fan of putting this variable into Constants, as it is not a constant. Why not injecting UserSettings here and checking the `userSettings.satteliteMap` like we do with `userSettings.convertToCelsius` and all the other settings?